### PR TITLE
Chairs now handle layer updates/rotation with setDir

### PR DESCRIPTION
**Pull request by ChangelingRain**
Removes a proc that did nearly nothing, too.

### CODE REVIEW

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 1]
```diff
@@ -78,31 +74,24 @@
 	return
 
 /obj/structure/chair/proc/handle_rotation(direction)
+	handle_layer()
 	if(has_buckled_mobs())
 		for(var/m in buckled_mobs)
 			var/mob/living/buckled_mob = m
-			buckled_mob.buckled = null //Temporary, so Move() succeeds.
-			if(!direction || !buckled_mob.Move(get_step(src, direction), direction))
```
Comment (ChangelingRain) [inline code/game/objects/structures/beds_chairs/chair.dm]: Line 33 was the only time this was called. Take a good look, then consider that chairs worked fine without the second part of the `||`.

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 2]
```diff
@@ -78,31 +74,24 @@
...
-				buckled_mob.buckled = src
-				setDir(buckled_mob.dir)
-				return 0
-			buckled_mob.buckled = src //Restoring
-	handle_layer()
```
Comment (ChangelingRain) [inline code/game/objects/structures/beds_chairs/chair.dm]: The only thing this proc actually did.

**Comment by phil235** [comment]
@optimumtact Stop merging things before the 24hr mark! This is the first time I'm seeing this PR and it's already merged...

**Comment by optimumtact** [comment]
Opened 13 June  3:24 PM NZST
Merged 14 June 3:24 PM NZST

**Pull request merged**